src/main/core/Listener/DeauthenticatedListener.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Claroline\CoreBundle\Listener;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Security\Http\Event\DeauthenticatedEvent;
+
+class DeauthenticatedListener
+{
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    public function onDeauthentication(DeauthenticatedEvent $deauthenticatedEvent)
+    {
+        $originalUser = $deauthenticatedEvent->getOriginalToken()->getUsername();
+        $refreshedUser = $deauthenticatedEvent->getRefreshedToken()->getUsername();
+
+        $this->logger->error(sprintf('Deauthentication on %s with %s', $originalUser, $refreshedUser));
+    }
```

Comment:
I'm not sure we want this: Symfony already logs a similar message on deauthentication (see ContextListener), and 
it's not an error. 
I think `SecurityEventSubscriber` should listen on this event and create a SecurityLog instead, as for others

Reply:
Yeah i totally missed to removed this file and add this event in `SecurityEventSubscriber` sorry, it's done :)